index.js

```diff
@@ -4,15 +4,9 @@ const Router = require('koa-router');
 const uuid = require('node-uuid');
 const AWS = require('aws-sdk');
 
-const S3_ACCESS_KEY = process.env.AWS_ACCESS_KEY_ID;
-const S3_SECRET_ACCESS_KEY = process.env.AWS_SECRET_ACCESS_KEY;
```

Comment:
Actually I think the README is ok. You can still set those environment variables, it's just not recommended in EC2/Lambda. Still useful in dev/test environments though.